src/test/java/com/firstwebapp/practice/spring/HelloWorldControllerTest.java

```diff
@@ -14,6 +14,6 @@
     public void testCapitalizeString(){
         assertEquals("This should be capitalized", "THIS SHOULD BE CAPITALIZED", HelloWorldController.capitalizeString("this should be capitalized"));
 
-        assertEquals("This should be hello world", "THIS SHOULD BE CAPITALIZED", HelloWorldController.capitalizeString("This should be hello world"));
+        assertEquals("This should be hello world", "THIS SHOULD BE HELLO WORLD", HelloWorldController.capitalizeString("This should be hello world"));

```

Comment:
This line seems irrelevant to what you're trying to accomplish in this lab. Make sure you're isolating your work in separate branches.